src/app/features/tasks/short-syntax.ts

```diff
@@ -198,7 +204,10 @@ const parseProjectChanges = (
 
     if (existingProject) {
       return {
-        title: task.title?.replace(`${CH_PRO}${projectTitle}`, '').trim(),
+        title: task.title
+          ?.replace(`${CH_PRO}${projectTitle}`, '')
+          .trim()
+          .replace('  ', ' '),
```

Comment:
The second `replace` here mirrors the behaviour on L229.

This is needed because the updated project name pattern matching is more specific about isolating only the project name itself.

As far as I understand, this replacement is intended to consume the whitespace character that delimits the project name from whatever follows (e.g. `task +proj 10m` becomes `task 10m` rather than `task  10m`).